scripts/ci/publish.ts

```diff
@@ -858,32 +858,21 @@ async function testPackages(
 ): Promise<void> {
   let order = flatten(publishOrder)
 
-  // If paralelism is set in builkite we split the testing
-  // Job 0 all but client
-  // Job 1 only client
+  // If parallelism is set in build-kite we split the testing
+  //  Job 0 - Node-API
+  //    PRISMA_CLIENT_ENGINE_TYPE="node-api"
+  //    PRISMA_CLI_QUERY_ENGINE_TYPE="node-api"
+  //  Job 1 - Binary
+  //    PRISMA_CLIENT_ENGINE_TYPE="binary"
+  //    PRISMA_CLI_QUERY_ENGINE_TYPE="binary"
   if (process.env.BUILDKITE_PARALLEL_JOB === '0') {
-    console.log(
-      'BUILDKITE_PARALLEL_JOB === 0 - running all tests excluding client',
-    )
-    const index = order.indexOf('@prisma/client')
-    if (index > -1) {
-      order.splice(index, 1)
-    }
+    console.log('BUILDKITE_PARALLEL_JOB === 0 - Node-API')
   } else if (process.env.BUILDKITE_PARALLEL_JOB === '1') {
-    console.log('BUILDKITE_PARALLEL_JOB === 1 - running client only')
-    order = ['@prisma/client']
+    console.log('BUILDKITE_PARALLEL_JOB === 1 - Binary')
   } else if (process.env.BUILDKITE_PARALLEL_JOB === '2') {
-    // This is to test Node-API
-    console.log(
-      'BUILDKITE_PARALLEL_JOB === 2 - running Node API tests for [sdk, migrate, client, cli, integration-tests]',
-    )
-    order = [
-      '@prisma/sdk',
-      '@prisma/migrate',
-      '@prisma/client',
-      'prisma',
-      '@prisma/integration-tests',
-    ]
+    // This is a tmp workaround
+    console.log('SKIPPING')
+    order = []
```

Comment:
Does that mean that this needs to be removed when we merge?

Reply:
I have removed this now that I know how to change the [buildkite parallelism](https://github.com/prisma/prisma/blob/main/.buildkite/test/test.yml#L3)